### ISSUE TITLE
Make progress graphic on the campaign page accessible

### ISSUE THREAD

**Issue by alicetragedy** [rails-girls-summer-of-code/summer-of-code#461]
As pointed out by @thegrumpygirl while reworking the copy for the [campaign page](https://railsgirlssummerofcode.org/campaign/), the graphic we use to show campaign progress is not accessible at all (there is no alt text). This makes it impossible for visually impaired people to have an idea of how many teams have been funded. 
It would be great to make changes towards something more accessible for everyone.

**Comment by alicetragedy** [comment]
Update: since we weren't able to do this for 2018, we're going to leave this on the back burner until we rework the site again.
